Edits to Git entries (11 total) (#562)
Codecademy/docs: content/git/concepts/push/push.md

  nothing to commit, working tree clean
  ```

> - Having a valid remote repository set, in this case there is remote repository called `origin`:

Author: I agree this sentence isn't great. I'm going to update it in both files for this PR, but I also think in the future adding an entry for `git remote` and linking to it here would be better.

  ```shell
  $ git remote -v

  origin    https://github.com/CompanyName/product-dev.git (fetch)
  origin    https://github.com/CompanyName/product-dev.git (push)
  ```

> At this stage, pushing is as simple as using the basic syntax with the correct remote and branch names:

Reviewer: ```suggestion
At this stage, pushing can accomplished with the names of a valid remote and branch:
```

Reviewer 2: can be accomplished*
